shape-the-future/redis.md

```diff
@@ -0,0 +1,15 @@
+---
+title: NServiceBus and Redis
+summary: Using NServiceBus with Redis
+reviewed: 2024-09-23
+---
+
+-- do we want an image here too?
+
+[Redis](https://redis.io/) provides cloud and on-prem solutions for caching, vector search, and NoSQL databases that seamlessly fit into any tech stack—making it simple for digital customers to build, scale, and deploy the fast apps our world runs on.
+
+Redis makes a good fit as a data store for the [Claim Check pattern](https://learn.microsoft.com/en-us/azure/architecture/patterns/claim-check) in NServiceBus.
+There's a [sample available that showcases this usage](/samples/databus/redis).
+
+Do you have additional use cases for Redis with NServiceBus?
+[Tell us more](https://github.com/Particular/NServiceBus/issues/6940)!
```

Comment:
I think we should lead the conversation with a few questions we would like answered as part of the feedback.

- Do you care about transactional integrity
- Is speed a factor in using Redis
- Do you currently use Redis, and if so, what for 
- Do you store business data in Redis
-